tasty-query/shared/src/main/scala/tastyquery/Symbols.scala

```diff
@@ -791,7 +801,7 @@ object Symbols {
         case Some(overloads) =>
           overloads.find {
             case decl: TermSymbol =>
-              decl.signature.exists(_ == overloaded.sig) && decl.targetName == overloaded.target
+              overloaded == decl.signedName
```

Comment:
this seems like it would fail for any parameterless def
```scala
package bar

class Bar:
  def bar = 1
  def bar(x: Int) = x

  def test =
    val b = new Bar
    b.bar
    b.bar(1)
```
in the above example `b.bar` is encoded with a `SELECTin`, this will have a signed name, so will resolve its symbol by a call to `getDecl` with a signed name, so delegating to this method with a signed name, thus it will never match

Reply:
actually dismiss this, I misread the TASTy, actually this would be a plain SELECT